go/integTest/standalone_commands_test.go

```diff
@@ -392,3 +392,16 @@ func (suite *GlideTestSuite) TestDBSize() {
 	assert.Nil(suite.T(), err)
 	assert.Greater(suite.T(), result, int64(0))
 }
+
+func (suite *GlideTestSuite) TestEchoWithOptionsWithRoute() {
+	client := suite.defaultClient()
+	options := options.NewEchoOptionsBuilder().
+		SetMessage("Hello").
+		SetRoute(config.SimpleNodeRoute(config.RandomRoute))
```

Comment:
Can we improve UX by the following way?
```suggestion
		SetRoute(config.RandomRoute)
```

Reply:
Already updated in my latest commit.